CONTRIBUTING.md

```diff
@@ -101,3 +101,9 @@ repository, we have a [community membership guide](./docs/governance/community_m
 
 - [How to write a good Git Commit message](https://chris.beams.io/posts/git-commit/) -
   Great way to make sure your Pull Requests get accepted.
+- **Log levels usage:**
```

Comment:
Please move this entire section it's own PR. Where possible, smaller PRs are better, especially PRs that touch on multiple aspects.

Here we have a PR that is about log levels in SDK Server, but also touches on contributing guidelines. It's very hard to approve / reject both.